libdnf/module/ModulePackageContainer.cpp

```diff
@@ -282,6 +282,11 @@ void ModulePackageContainer::createConflictsBetweenStreams()
     }
 }
 
+bool ModulePackageContainer::empty()
```

Comment:
This method does not change the state of object and does not generate exception. So I prefer: 
`bool ModulePackageContainer::empty() const noexcept`